components/brave_welcome_ui/state/component_types.ts

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) 2022 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at https://mozilla.org/MPL/2.0/.
+
+import WebAnimationPlayer from '../api/web_animation_player'
+
+export enum ViewType {
+  Default, // the welcome ui
+  SelectBrowser,
+  SelectProfile,
```

Comment:
These are both Import related so perhaps also give them the Import prefix

Reply:
done